charts/sysdig/templates/cspmCollector/deployment-cspm-collector.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.cspmCollector.deploy }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sysdig.fullname" . }}-cspm-collector
+  labels:
+    app.kubernetes.io/name: {{ include "sysdig.name" . }}-cspm-collector
+{{ include "sysdig.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.cspmCollector.settings.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "sysdig.name" . }}-cspm-collector
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "sysdig.name" . }}-cspm-collector
+        app.kubernetes.io/instance: {{ .Release.Name }}
+{{ include "sysdig.labels" . | indent 8 }}
+    spec:
+      serviceAccountName: {{ template "sysdig.serviceAccountName" .}}
```

Comment:
following on from the other comment, I think it may make sense to have a separate service account for this deployment. This will make it easier to split out later when moving these changes to the node-analyzer chart

Reply:
Agree. I changed it so that the cspm-collector has its own service account.